package.json

```diff
@@ -35,15 +35,15 @@
     "validate-commits": "./bin/devkit-admin validate-commits",
     "prepush": "node ./bin/devkit-admin hooks/pre-push",
     "webdriver-update-appveyor": "webdriver-manager update --standalone false --gecko false --versions.chrome 2.37",
-    "webdriver-update-circleci": "webdriver-manager update --standalone false --gecko false --versions.chrome 2.33"
+    "webdriver-update-circleci": "webdriver-manager update --standalone false --gecko false --versions.chrome $CHROMEDRIVER_VERSION_ARG "
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/angular/angular-cli.git"
   },
   "engines": {
-    "node": ">= 8.9.0",
-    "npm": ">= 5.5.1"
+    "node": ">=10.3.0 <11.0.0",
```

Comment:
This may confuse people looking at the repo.  Do we require 10 to build/develop?

Reply:
With this PR, we run all tests on Node 10 and run only test running the CLI on Node 8. So developing on node 8 is a bit a at-your-own-risk kind of thing.

I think yarn will yell at you because it actually bothers to check the engines properly so I'm not very opposed to leaving it at `"node": ">= 8.9.0",` to stop that. But it is misleading.

Reply:
This line is propagated to all `package.json` from the build script.

Reply:
Followup PR on https://github.com/angular/angular-cli/pull/12202.